lua/weapons/weapon_peacekeeper.lua

```diff
@@ -103,24 +103,20 @@ function SWEP:Deploy()
 end
 
 function SWEP:SprintMod()
-	return ((math.cos(((self.SprintNess or 0)+1)*math.pi)+1)*0.5)
-end
-
-function SWEP:IsSprinting()
-	return self.Owner:GetVelocity():Length() > (self.Owner:GetWalkSpeed()+self.Owner:GetRunSpeed())*0.5
+	return ((math.cos((1)*math.pi)+1)*0.5)
```

Comment:
can you clean this up properly? if you have no sprinting behavior, there shouldn't be anything sprint-related in the file

Reply:
this is for weapon swaying while running

Reply:
it returns the same value no matter what though, it doesn't do anything. whatever it returns can just be a constant value where it is called. The weapon swaying while sprinting is a built-in thing.

Reply:
I can't believe I didn't notice that lol